lib/identity_cache/cache_hash.rb

```diff
@@ -1,16 +1,23 @@
-# Use CityHash for fast hashing if it is available; use Digest::MD5 otherwise
-begin
-  require 'cityhash'
-rescue LoadError
+# Use a hashing library for fast hashing if it is available; use Digest::MD5 otherwise
+AVAILABLE_HASHING_GEMS = %w{cityhash xxhash murmurhash3}
```

Comment:
This sill have the same problem tho, I mean is better because it uses the same as before but the fact that a xxhash user can unwittingly endup using cityhash remains.

I'm ok with the idea of this, but if we want it in the code base it should be more explicit.